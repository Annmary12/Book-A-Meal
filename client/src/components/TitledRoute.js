import React from 'react';
import PropTypes from 'prop-types';
import { Route } from 'react-router';
import DocumentTitle from 'react-document-title';

const TitledRoute = props => (
  <DocumentTitle title={`${props.title} - Book-A-Meal`}>
    <Route {...props} />
  </DocumentTitle>
);

TitledRoute.propTypes = {
  title: PropTypes.string
};

TitledRoute.defaultProps = {
  title: ''
};

export default TitledRoute;
