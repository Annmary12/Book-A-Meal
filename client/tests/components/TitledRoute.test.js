import React from 'react';
import TitledRoute from '../../src/components/TitledRoute';

describe('TitledRoute component', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<TitledRoute />);

    expect(wrapper).toMatchSnapshot();
  })
})