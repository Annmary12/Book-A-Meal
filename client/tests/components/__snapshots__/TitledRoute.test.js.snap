// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TitledRoute component should match snapshot 1`] = `
<SideEffect(DocumentTitle)
  title=" - Book-A-Meal"
>
  <Route
    title=""
  />
</SideEffect(DocumentTitle)>
`;
